pkg/client/dart/templates/api.dart

```diff
@@ -32,10 +32,14 @@ class {{ $elem.Name }} {
 {{- 	end }}
   }{{ else }};{{ end }}
 
-  Map<String, dynamic> getRequestObject(Map<String, dynamic> obj, List<String> routingPath) {
+  Map<String, dynamic> getRequestObject(Map<String, dynamic> obj, List<String> routingPath, bool isGET) {
     final copied = {...obj};
-    
-    copied.removeWhere((key, value) => routingPath.contains(key));
+
+    copied.forEach((key, value) {
+      if (routingPath.contains(key))
+        copied.remove(key);
+      else if (isGET) copied[key] = value.toString();
```

Comment:
上改行してるのでここも改行したいかもしれない
```suggestion
      else if (isGET)
        copied[key] = value.toString();
```